packages/cli-tool/src/add-ons/ui-framework/index.ts

```diff
@@ -0,0 +1,23 @@
+import { cli } from 'cli-ux';
+
+import setupBootstrap from './bootstrap';
+
+export const UI_FRAMEWORK_OPTIONS: { [key: string]: string } = {
+  bootstrap: 'Bootstrap',
+  tailwind: 'Tailwind CSS',
+  none: 'None',
+};
+
+export const setUIFramework = async(
+  appName: string,
+  uiFramework: string,
+): Promise<void> => {
+  if (uiFramework === 'bootstrap') {
+    cli.info('Configure Bootstrap...');
+
+    await setupBootstrap(appName);
```

Comment:
The await does not propagate (and I think that's a good thing, it means you can use an AWAIT inside an async func and still run this function asynchronously in a high-level code — if needed).

Here is a way to demonstrate it:
1. Add log after `await setupBootstrap(appName);`, we call this log "This should run AFTER."
2. Add a log at the last step of the setupBootstrap function (before the last promise get resolved). We call this "This should run before..."

| With AWAIT | Without AWAIT |
|---|---|
|<img width="375" alt="image" src="https://user-images.githubusercontent.com/77609814/177911249-60892154-8ea1-481b-baac-356d2196d9e6.png">|<img width="1128" alt="image" src="https://user-images.githubusercontent.com/77609814/177911425-1bb266a7-ee17-45b9-80dd-0b1198205202.png">|
| Work as expected | It did not even show the "this should run BEFORE, because the "clean" function has run before the end of the bootstrap setup, so the `.add-ons` folder was deleted before ^^ |

Reply:
Update: @nvminhtue explained to me the issue, it is that we have an `awaited function running awaited functions` (thus creating a chain of abstracted threads). 
I've resolved it in 4ea518e. Now running some tests  ~

Reply:
+ fix some linting issues 🙈  d98b871